cmd/fyne/internal/commands/build.go

```diff
@@ -55,12 +58,32 @@ func (b *builder) build() error {
 	if goos != "ios" && goos != "android" && goos != "wasm" {
 		env = append(env, "GOOS="+goos)
 	} else if goos == "wasm" {
+		versionConstraint = version.NewConstrainGroupFromString(">=1.17")
```

Comment:
As it's only used in this instance would it not be easier to just run the check here?

Reply:
I am expecting to set the test to <=1.17 for gopherjs when I add it here. That's why I kept it as a general feature.

Reply:
Fair enough